README.md

```diff
@@ -284,6 +284,12 @@ A number of options are available to allow you to customise the SDK:
   smsNumberCountryCode: 'US'
   ```
 
+- **`applicantDetails {Object} optional`**
+  Some applicant details can be specified ahead of time, so that the applicant doesn't need to fill them in themselves.
+
+  The following details can be used by the SDK:
+    - `smsNumber` (optional) : The applicant's mobile number, which can be used for sending any SMS messages to the applicant. This should be formatted as a string, with a country code (e.g. `"+447500123456"`)
```

Comment:
Can we also add. "An example SMS message sent by the SDK is when a user requests to use their mobile devices to take photos." Just to give more context.